### PR TITLE
Hide test metrics for now

### PR DESCRIPTION
**Pull request by Robert-Krajcik**
test metrics tab removed from dashboard and from report preview
image

### CODE REVIEW

**Review by AARON-CLARK — CHANGES_REQUESTED** [review]
There appears to be conflicts between this branch and master.

**Comment by Robert-Krajcik** [comment]
OK updated conflict with report_doc.Rmd

File: Reports/Report_doc.Rmd [hunk 1]
```diff
@@ -289,10 +289,10 @@ HTML(display_comments(comment_cum))
 
 \newpage
```
Comment (AARON-CLARK) [inline Reports/Report_doc.Rmd:290]: Without testing metrics we could remove this `\newpage`. Right now, it is producing an extra page for the word doc version of the report:

image

**Review by AARON-CLARK — CHANGES_REQUESTED** [review]

File: Reports/Report_doc.Rmd (hunk 1 above)
Comment (Robert-Krajcik) [reply Reports/Report_doc.Rmd]: I will comment it out now

**Review by AARON-CLARK — APPROVED** [review]

**Pull request merged**